find.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/sh
+#!/bin/bash
+set -euo pipefail
+shopt -s nullglob
```

Comment:
Love it.

Reply:
lol, I had to remove it... when I set this the shell word disappears and `ls -1r` is run, which just lists everything in the top level directory. I've removed `ls -1r`; is there any reason you were listing the files in reverse? If older corepops work, why not use them?

Reply:
I was following a recommendation. But I have decided that the order needs to be curated, so I will be removing that very soon.